client/a8c-for-agencies/style.scss

```diff
@@ -39,6 +39,10 @@
 }
 
 .theme-a8c-for-agencies {
+	.button {
+		border-radius: 4px;
+	}
```

Comment:
Ensures that all our A4A buttons have rounded corners according to the design spec.